release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -272,6 +272,16 @@ playbooks.
 [[ocp-311-storage]]
 === Storage
 
+[[ocp-311-openshift-container-storage]]
+==== OpenShift Container Storage ====
```

Comment:
You can delete the trailing ==== after the heading

Reply:
Oops, thanks!